src/main/java/cucumber/pageObjects/NavigationShared.java

```diff
@@ -137,6 +137,7 @@ public void accessLoginPage(String publicBureau, String env) {
         System.setProperty("env.database", env.toLowerCase());
         System.setProperty("envName", env.toLowerCase());
 
+        driver.manage().deleteAllCookies();
```

Comment:
Why have you added this here? I wanted to only add it to the specific cookies test steps, not this one